README.md

```diff
@@ -1,52 +1,127 @@
+#### Architectures
 
-* arm32v7 [![Build Status](https://doi-janky.infosiftr.net/buildStatus/icon?job=multiarch/arm32v7/gazebo)](https://doi-janky.infosiftr.net/job/multiarch/job/arm32v7/job/gazebo/)
+| Type | Status |
+|---|---|
+| [amd64](https://hub.docker.com/r/amd64/gazebo/tags) | [![Build Status](https://doi-janky.infosiftr.net/buildStatus/icon?job=multiarch/amd64/gazebo)](https://doi-janky.infosiftr.net/job/multiarch/job/amd64/job/gazebo/) |
+| [arm32v7](https://hub.docker.com/r/arm32v7/gazebo/tags) | [![Build Status](https://doi-janky.infosiftr.net/buildStatus/icon?job=multiarch/arm32v7/gazebo)](https://doi-janky.infosiftr.net/job/multiarch/job/arm32v7/job/gazebo/) |
+| [arm64v8](https://hub.docker.com/r/arm64v8/gazebo/tags) | [![Build Status](https://doi-janky.infosiftr.net/buildStatus/icon?job=multiarch/arm64v8/gazebo)](https://doi-janky.infosiftr.net/job/multiarch/job/arm64v8/job/gazebo/) |
 
+---
 
 ## [OSRF Profile](https://hub.docker.com/u/osrf/)
-### [ROS ![Docker Pulls](https://img.shields.io/docker/pulls/osrf/ros.svg) ![Docker Stars](https://img.shields.io/docker/stars/osrf/ros.svg)](https://registry.hub.docker.com/u/osrf/ros/)
-[![Compare Images](https://images.microbadger.com/badges/image/osrf/ros.svg)](https://microbadger.com/#/images/osrf/ros)
 
-* amd64 [![Docker Build Status](https://img.shields.io/docker/build/osrf/ros.svg)](https://hub.docker.com/r/osrf/ros/builds/)
+The following repos are hosted under the OSRF Organization on Docker Hub. These images are tagged for development and advanced use cases. If you require extra dependencies not available from the Official Library images above, then these using these images may help save you build time during continuous integration or local development.
+
+### [ROS ![Docker Pulls](https://img.shields.io/docker/pulls/osrf/ros.svg?label=pulls) ![Docker Stars](https://img.shields.io/docker/stars/osrf/ros.svg?label=stars)](https://hub.docker.com/r/osrf/ros/)
 
+This repo builds `FROM` the Official Library by including additional meta-packages such for desktop installations. These images may be helpful using containers for development or when display forwarding when using GUIs.
+
+#### Repo Info
+[![Compare Images](https://images.microbadger.com/badges/image/osrf/ros.svg)](https://microbadger.com/#/images/osrf/ros)
+
+- `nightly`
+  - _Description:_
+    - includes pre-installed environment from nightly job on the build farm
+    - closely mimics the Official Library images as be interchangeable
+  - _Purpose:_
+    - intended for CI and testing against the latest nightly builds
+    - early detection of regressions or deprications from upstream
+- `nightly-rmw`
+  - _Description:_
+    - builds `FROM` `nightly` and installs open source rmw libraries
+    - including non default vendors (e.g. Opensplice, etc..)
```

Comment:
I'd rather be exact in these lists so that users know what they get. And update it if the list changes in the future (same below)

```suggestion
    - including non default vendors (currently Opensplice)
```